src/Raven.Server/Documents/Queries/Highlightings/HighlightingField.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Raven.Server.Documents.Queries.Highlightings
 {
-    public sealed class HighlightingField
```

Comment:
Can we project a field Content (without highlighting) and highlight Content at the same time?

Reply:
Yes. Because we essentially project it only if it wasnt projected before. Having said that, I really dislike the solution but that is how Lucene works.